stl/inc/strstream

```diff
@@ -34,7 +34,12 @@ public:
     }; // set if character array ownership given away
     using _Strstate = int;
 
-    explicit __CLR_OR_THIS_CALL strstreambuf(streamsize _Count = 0) {
+    // construct with empty character array

```

Comment:
Conventionally this comment should attach to the function's open brace:
```c++
__CLR_OR_THIS_CALL strstreambuf() { // construct with empty character array
```
or wrap to the next line if it won't fit on the same line as the open brace:
```c++
__CLR_OR_THIS_CALL strstreambuf() {
    // construct with empty character array
```

Reply:
Done